MoltenVK/MoltenVK/GPUObjects/MVKImage.h

```diff
@@ -566,6 +582,10 @@ class MVKImageViewPlane : public MVKBaseDeviceObject {
     bool _useMTLTextureView;
 	bool _useNativeSwizzle;
 	bool _useShaderSwizzle;
+
+    // For now, this is only used for VK_EXT_image_2d_view_of_3d. Specifically, when the backing image of this view is 3D and the view is 2D,
+    // this texture will be allocated using a placement heap "on top of" the 3D textures backing memory.
+    id<MTLTexture> _mtlShadowTexture;
```

Comment:
Since `_mtlTexture` end up holding a reference to the same object, does this need to be a separate iVar than `_mtlTexture`? 

If it is needed for some reason, move it up just below the `_mtlTexture` declaration. We try to organize iVars to minimize gaps in the underlying class structure (ie- keep all 8-byte iVars together).

It also feels dangerous that the `MTLTexture` is not retained in both `_mtlTexture` and `_mtlShadowTexture`, and released from both in the destructor. If in the future, we might possibly have different objects in the two iVars, then we'll end up with a hard to discover memory leak.

Finally, maybe it's me, but when I see `_mtlShadowTexture`, I keep thinking "shadow map texture". Maybe rename it to `_mtlAliasTexture` or something.

Reply:
Nope, there is no need for the extra MTLTexture in MVKImageViewPlane; I will remove it. I added this from a debug session and never reverted it (and also forgot to clean it up in the destructor when I was using it :/).